cmd/siderolink-agent/main.go

```diff
@@ -31,9 +32,14 @@ func main() {
 }
 
 func run(ctx context.Context) error {
+	logger, err := zap.NewDevelopment()
```

Comment:
Should we be using the Development logger in regular code?  Though realistically, I've never really understood why they're called Development and Production; they seem to just be different presets.

Reply:
this is a test code specifically, it's `main.go` of this test repository, it's used as a library in Sidero without this main package